typescript/sdk/src/deploy/core/types.ts

```diff
@@ -12,6 +24,7 @@ export type MultisigIsmConfig = {
 export type CoreConfig = {
   multisigIsm: MultisigIsmConfig;
   owner: types.Address;
+  igp: InterchainGasPaymasterConfig;
```

Comment:
Again, out of scope likely, but I wonder if it should not be part of core, but a separate "app". cc @yorhodes